requirements.txt

```diff
@@ -0,0 +1,2 @@
+numpy
+matplotlib
```

Comment:
Can you also add `Pillow` to the list? This is needed during image export.

Reply:
As there was no problem in the test, when installing matplotlib, it is assumed that Pillow was not installed together.
However, I explicitly added Pillow. Commit: 57cd9d3973d20d932ba2722704d79d0d514d731c